src/engine/renderer/glsl_source/processSurfaces_cp.glsl

```diff
@@ -77,13 +77,36 @@ uniform uint u_ViewID;
 uniform uint u_SurfaceCommandsOffset;
 uniform uint u_CulledCommandsOffset;
 
+#if defined(HAVE_GL_KHR_shader_subgroup_basic) && defined(HAVE_GL_KHR_shader_subgroup_arithmetic)
```

Comment:
These feature names don't seem to match up. There's no "GL" in the ones in the previous commit

Reply:
Fixed.

Reply:
Now one is missing the HAVE

Reply:
Oh, right, fixed now.